### ISSUE TITLE
-

### ISSUE THREAD

**Issue by SuleymanCetiner** [combust/mleap-demo#19]
Hello there,

I have an issue for running just a basic script : 

from river import linear_model
from river import metrics
from river import evaluate
from river import preprocessing
import pandas as pd

data = pd.read_csv("C:/Users/Monster/Desktop/LveR.csv")

# Import label encoder
from sklearn import preprocessing
from sklearn.model_selection import train_test_split
# label_encoder object knows how to understand word labels.
label_encoder = preprocessing.LabelEncoder()

# Encode labels in column 'species'.
data['Class'] = label_encoder.fit_transform(data['Class'])

data['Class'].unique()

X=data.iloc[:,:-1]
y=data.iloc[:,-1]

X_train,X_test,y_train,y_test = train_test_split(X,y,random_state = 11,test_size=0.25,shuffle=True)

model = (
    preprocessing.StandardScaler() |
    linear_model.LogisticRegression())

metric=metrics.ROCAUC()
evaluate.progressive_val_score(X_test,y_test,model,metric)

## That says " Pipeline object has no attribute "works_with". I get confused. Any type of help will be appreciated. Thanks ! 

![Error ](https://user-images.githubusercontent.com/38746955/193858901-a1c979e3-89d7-45f3-892a-dfea8066ef06.png)
